src/components/ng-redux.ts

```diff
@@ -70,12 +72,18 @@ export class NgRedux<RootState> {
             = <Redux.StoreEnhancerStoreCreator<RootState>>compose(
                 applyMiddleware(...middleware),
                 ...enhancers
-                )(createStore);
+            )(createStore);
         const store = finalCreateStore(reducer, initState);
 
         this._store = store;
-        this._store$ = new BehaviorSubject(store.getState());
-        this._store.subscribe(() => this._store$.next(this._store.getState()));
+        let x = store as any;
+        
+        this._store$ = Observable.from(x); //new BehaviorSubject(store.getState());
```

Comment:
Weird -  I wonder if there's a subtle different in 'zoning' between the two ways of creating observables.